internal/sdkv2provider/schema_cloudflare_access_application.go

```diff
@@ -54,13 +54,53 @@ func resourceCloudflareAccessApplicationSchema() map[string]*schema.Schema {
 				return oldValue == newValue
 			},
 		},
+		"domain_type": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validation.StringInSlice([]string{"public", "private"}, false),
+			Description:  fmt.Sprintf("The type of the primary domain. %s", renderAvailableDocumentationValuesStringSlice([]string{"public", "private"})),
+			DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+				appType := d.Get("type").(string)
+				// Suppress the diff if it's an app type that doesn't need a `domain` value.
+				if appType == "infrastructure" {
+					return true
+				}
+
+				return oldValue == newValue
+			},
+		},
+		"destinations": {
+			Type:          schema.TypeList,
+			Optional:      true,
```

Comment:
Should this be `Computed` as well? It's unclear what that actually does 🧐

Reply:
https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed

Reply:
Looks like it probably shouldn't be computed then, thanks!